packages/contracts/src/libraries/GameErrorsLib.sol

```diff
@@ -0,0 +1,105 @@
+// SPDX-License-Identifier: BSD-3-Clause-Clear
+pragma solidity ^0.8.0;
+
+library GameErrorsLib {
+    // ERRORS
+
+    // TODO This is an error from the inventory contract, but we need to reproduce it here
+    //      so that it is included in the ABI, which allows the frontend to parse the error.
+    //      We need to figure out a way to systematically import all errors that we may revert with
+    //      from other contracts.
```

Comment:
Did you check to see if this problem is present after separating out the files? If not I'll have a look on how to check.

Reply:
The error is included in the generated ABI for the inventory contract. I assuming you mean the auto generated ABI in `generated.ts`.